### DISCUSSION TITLE
TTS utf-8 input

### DISCUSSION THREAD

**Discussion by oytunturk** [NVIDIA-NeMo/NeMo#3499] [Q&A] [answered]
Do TTS training and inference recipes support utf-8 encoded character sets/input text? Or, is there another preferred encoding for languages other than English? Any suggestions on what to pay attention to in input text and in text normalization when using Nemo in other languages?

Thanks!

**Comment by XuesongYang** [comment, marked as answer]
Yes, we support it now. Please see 
* German character tokenizer: https://github.com/NVIDIA/NeMo/blob/main/nemo/collections/tts/torch/tts_tokenizers.py#L219
* Tutorial of German TTS: https://github.com/NVIDIA/NeMo/blob/main/tutorials/tts/Fastpitch_Training_GermanTTS.ipynb
